types/xdr.d.ts

```diff
@@ -353,9 +353,33 @@ declare namespace xdr {
       | 'manageDatum'
       | 'bumpSequence'
       | 'manageBuyOffer'
-      | 'pathPaymentStrictSend';
+      | 'pathPaymentStrictSend'
+      | 'createClaimableBalance'
+      | 'claimClaimableBalance'
+      | 'beginSponsoringFutureReserf'
```

Comment:
oh -- this is weird, it should say `beingSponsoringFutureReserves` -- I'll check what's going on